### PR TITLE
Further iterate on Migrate spec

### PR DESCRIPTION
**Pull request by janpio**
part of #156

### CODE REVIEW

File: lift/Readme.md [hunk 1]
```diff
@@ -98,8 +97,7 @@ This is error-prone and stressful, especially when you're operating on your prod
 
 ## The Lift Approach
 
-Prisma's Lift works differently. While Lift still has a `migrations/` folder, the migrations are generated for you. With Lift, you just need to change your
-`schema.prisma` file and run `lift save`. This will generate the necessary steps to transition your schema from A to B.
+Prisma's Lift works differently. While Lift still has a `migrations/` folder, the migrations are generated for you. With Lift, you just need to change your project's `schema.prisma` file and run `lift save`. This will generate the necessary steps to transition your schema from A to B.
 
 A result might look like this:
```
Comment (janpio) [inline lift/Readme.md]: The result of what exactly is this actually @matthewmueller? The previous sentence makes you think these could be the generated steps, but I don't think that is the case, right?

File: lift/Readme.md [hunk 2]
```diff
@@ -233,10 +177,8 @@ the **Migration History**. It includes all successful and failed migrations of t
...
 ## Architecture
 
-Lift has 2 parts: a **Lift Client** and a **Lift Engine**. By default, the Lift Engine runs locally as a
-[sidecar process](https://blog.davemdavis.net/2018/03/13/the-sidecar-pattern/). This makes it easy to get started. As your team's data requirements become more
-complex, you may prefer to handle your migrations on a remote host where you have fine-grained access control over migrations. We will make this possible in the
-future.
+Lift has 2 parts: a **Lift Client** and a **Lift Engine**. The Lift Engine runs locally as a
+[sidecar process](https://blog.davemdavis.net/2018/03/13/the-sidecar-pattern/) to the Lift Client. 
 
 ```
```
Comment (janpio) [inline lift/Readme.md]: Can we make this diagram align a bit better and have straight lines @matthewmueller?

File: lift/Readme.md (hunk 1 above)
Comment (matthewmueller) [reply lift/Readme.md]: > The result of what exactly is this actually @matthewmueller?

I don't know what this means haha.

The steps are generated for you, sometimes you need to edit them by hand though (renaming a model).

File: lift/Readme.md (hunk 2 above)
Comment (matthewmueller) [reply lift/Readme.md]: You mean center-aligned?

Which lines aren't straight, the arrows? Or are you talking about the line segments not being connected?

File: lift/Readme.md (hunk 1 above)
Comment (janpio) [reply lift/Readme.md]: So what do https://github.com/prisma/specs/pull/195/files#diff-8302b741093dd01fdda810da584dac22L106-L130 actually show here? What is this - it looks like a diff of a model part of a schema. But why is this here? How does it relate to Lift?

Comment (janpio) [reply lift/Readme.md]: Ping @matthewmueller

**Pull request merged**